python_src/src/lamp_py/migrations/versions/performance_manager_prod/001_5d9a7ee21ae5_initial_prod_schema.py

```diff
@@ -0,0 +1,966 @@
+    """
+    op.execute(create_static_trips_create_branch_trunk)
+
+    create_rt_braintree_function = """
+        CREATE OR REPLACE FUNCTION rt_red_is_braintree_branch(p_trip_id int) RETURNS boolean AS $$ 
+        DECLARE
+            found_check bool;
+        BEGIN 
+            SELECT
+                true
+            INTO
+                found_check
+            FROM 
+                vehicle_events ve
+            WHERE 
+                ve.pm_trip_id = p_trip_id
+                AND ve.stop_id IN ('70097', '70098', '70099', '70100', '70101', '70102', '70103', '70104', '70105', '70095', '70096')
+            LIMIT 1
+            ;
+            IF FOUND THEN
+                RETURN true;
+            ELSE
+                RETURN false;
+            END IF;
+        END;
+        $$ LANGUAGE plpgsql;
+    """
+    op.execute(create_rt_braintree_function)
+
+    create_rt_ashmont_function = """
+        CREATE OR REPLACE FUNCTION rt_red_is_ashmont_branch(p_trip_id int) RETURNS boolean AS $$ 
+        DECLARE
+            found_check bool;
+        BEGIN 
+            SELECT
+                true
+            INTO
+                found_check
+            FROM 
+                vehicle_events ve
+            WHERE 
+                ve.pm_trip_id = p_trip_id
+                AND ve.stop_id IN ('70087', '70088', '70089', '70090', '70091', '70092', '70093', '70094', '70085', '70086')
+            LIMIT 1
+            ;
+            IF FOUND THEN
+                RETURN true;
+            ELSE
+                RETURN false;
+            END IF;
+        END;
+        $$ LANGUAGE plpgsql;
+    """
+    op.execute(create_rt_ashmont_function)
+
+    create_get_rt_branch_trunk_id = """
+        CREATE OR REPLACE FUNCTION update_rt_branch_trunk_id() RETURNS TRIGGER AS $$ 
+        BEGIN 
+            IF NEW.route_id ~ 'Green-' THEN
+                NEW.branch_route_id := NEW.route_id;
+                NEW.trunk_route_id := 'Green';
+            ELSEIF NEW.route_id = 'Red' THEN
+                IF rt_red_is_braintree_branch(NEW.pm_trip_id) THEN
+                    NEW.branch_route_id := 'Red-B';
+                ELSEIF rt_red_is_ashmont_branch(NEW.pm_trip_id) THEN
+                    NEW.branch_route_id := 'Red-A';
+                ELSE
+                    NEW.branch_route_id := null;
+                END IF;
+                NEW.trunk_route_id := NEW.route_id;
+            ELSE
+                NEW.branch_route_id := null;
+                NEW.trunk_route_id := NEW.route_id;
+            END IF;
+            RETURN NEW;
+        END;
+        $$ LANGUAGE plpgsql;
+    """
+    op.execute(create_get_rt_branch_trunk_id)
+
+    create_trigger = """
+        CREATE TRIGGER rt_trips_update_branch_trunk BEFORE UPDATE ON vehicle_trips 
+        FOR EACH ROW EXECUTE PROCEDURE update_rt_branch_trunk_id();
+    """
+    op.execute(create_trigger)
+
+    create_service_id_by_date_and_route = """
```

Comment:
all of these are just direct reproductions?

Reply:
They are direct reproductions of the most recent definitions from the old migration files.